Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -519,7 +521,7 @@ namespace OpenInfraPlatform {
 						// convert the control points
 						std::vector<carve::geom::vector<3>> splinePoints = convertIfcCartesianPointVector(bspline_curve->ControlPointsList);
 
-						SplineConverterT<typename IfcEntityTypesT>::convertIfcBSplineCurve(bspline_curve, splinePoints, targetVec);
+						splineConverter->convertIfcBSplineCurve(bspline_curve, splinePoints, targetVec);
 						return;
 
 						// TO DO: Implement IfcBSplineCurveWithKnots

```

Comment:
Please delete these TODOs, if this is done with the convertBSplineCurve function.

Reply:
Done in commit ba0ea5d